src/Core/Abstractions/IApiService.cs

```diff
@@ -84,7 +84,9 @@ public interface IApiService
         Task<SendResponse> PutSendRemovePasswordAsync(string id);
         Task DeleteSendAsync(string id);
         Task<PasswordlessLoginResponse> GetAuthRequestAsync(string id);
+        Task<PasswordlessLoginResponse> GetAuthResponseAsync(string id, string accessCode);
         Task<PasswordlessLoginResponse> PutAuthRequestAsync(string id, string key, string masterPasswordHash, string deviceIdentifier, bool requestApproved);
+        Task<PasswordlessLoginResponse> PostCreateRequestAsync(string email, string publicKey, string deviceIdentifier, string accessCode, AuthRequestType? type, string fingerprintPhrase);
```

Comment:
Change the signature to have one only parameter for the request `PasswordlessCreateLoginRequest`. It's simpler to refactor afterwards if another parameter is needed or removed or modified, you just simply change the model of the request and you don't have to change the signatures.